src/main/java/net/openhft/chronicle/core/Mocker.java

```diff
@@ -21,11 +21,8 @@
 import org.jetbrains.annotations.NotNull;
 
 import java.io.PrintStream;
-import java.io.PrintWriter;
 import java.io.StringWriter;
 import java.util.concurrent.BlockingQueue;
-import java.util.function.BiConsumer;
-import java.util.function.Consumer;
 
 /**
  * The Mocker class provides utility methods for creating mocked instances of interfaces.
```

Comment:
should refer to util.Mocker with a comment explaining this here left for backwards compatibility?

Reply:
I have added a comment